### PR TITLE
Fix to open ADS release notes instead of VS Code

### LINKED ISSUE #7242: Insiders builds show vscode release notes after upgrade
- Azure Data Studio Version: `master`

Steps to Reproduce:

1. Update ADS insiders build

Expected: No release notes or correct release notes

Actual: vscode release notes

image

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Fixes #7242

### CODE REVIEW

**Comment by anthonydresser** [comment]
Not sure we want to open the browser every time ads is updated, which would be the behavior here? What was the previous behavior?

**Comment by Charles-Gagnon** [comment]
Correct. Previous behavior was to open VS Code release notes (which is a lot worse because it'd open it for a version like 1.12 which was from 2 years ago).

It's fairly normal for programs to open up web browsers with release notes so I don't think it'd be too bad - although I'd need to check if this would do that for insiders builds updates as well since that'd be a bit much I think (and provides no useful information).

Otherwise just having this no-op for now is fine too if we don't want to invest into fixing it to pull the actual release notes from the web.

**Comment by Charles-Gagnon** [comment]
There's also logic for prompting the user if they want to open release notes we could use as well. Although given that users can always disable showing release notes in startup I don't really see a huge problem with having it open the release notes in the browser. 

image

**Comment by kburtram** [comment]
I think showing an update notification with a button to open the release notes is less intrusive.  I thought that was the behavior we currently have for stable builds.

**Comment by Charles-Gagnon** [comment]
Alright, went back to that behavior. Still kept the check for not showing on insiders since there's currently nothing useful that would do.

screen

**Comment by Charles-Gagnon** [comment]
@kburtram @anthonydresser Are you satisfied with the current behavior?

File: src/vs/platform/update/common/update.config.contribution.ts [hunk 1]
```diff
@@ -48,7 +48,7 @@ configurationRegistry.registerConfiguration({
 			type: 'boolean',
 			default: true,
 			scope: ConfigurationScope.APPLICATION,
-			description: localize('showReleaseNotes', "Show Release Notes after an update. The Release Notes are fetched from a Microsoft online service."),
+			description: localize('showReleaseNotes', "Show Release Notes after an update. The Release Notes are opened in a new web browser window."), // {{SQL CARBON EDIT}} Update text to be correct for ADS
```
Comment (abist) [inline src/vs/platform/update/common/update.config.contribution.ts]: we should change the localize key if you're changing the text for it

**Comment by abist** [comment]
@Charles-Gagnon This merge had a failing build here - https://dev.azure.com/azuredatastudio/azuredatastudio/_build?definitionId=6&_a=summary but I'm not sure what this build is for

File: src/vs/platform/update/common/update.config.contribution.ts (hunk 1 above)
Comment (Charles-Gagnon) [reply src/vs/platform/update/common/update.config.contribution.ts]: Good catch, thanks!

**Pull request merged**